HISTORY.md

```diff
@@ -3,6 +3,9 @@
 ### Performance Improvements
 * Fixed an iterator performance regression for delete range users when scanning through a consecutive sequence of range tombstones (#10877).
 
+### Bug fix
+* Fixed a bug cased by `DB::SyncWAL()` on a obsoleted WAL and recording such WAL addition of an obsoleted WAL in a new manifest, affecting `track_and_verify_wals_in_manifest`. Without the fix, application may see "open error: Corruption: Missing WAL with log number" while trying to open the db. The corruption is a false alarm but prevents DB open (#10892).
```

Comment:
Too much implementation detail IMO. How about: "For users of `track_and_verify_wals_in_manifest`, fixed a race condition in MANIFEST rollover (see `max_manifest_file_size`) leading to false positive `DB::Open()` failures"?

Reply:
Fixed